### PR TITLE
CL fixes

### PR DESCRIPTION
**Pull request by jatinmore**
# Changes Done
* added working installation links
* badge icons size fix
* card with dismiss fix
* stack list added
* heading classes added

### CODE REVIEW

**Comment by athar05** [comment]
Overall the UI looks awesome. 

Just a suggestion for the badge with icon component. 

You can consider giving the icons a background color for better aesthetics. 

image

**Review by rahul31agrawal — APPROVED** [review]
Hi Jatin,
I have a optional suggestion, In stacked list example  
1. the icon & the text should be in one line
2. The overall box should be a bit smaller, then it will look good
Rest looks good to merge
Thanks

**Comment by athar05** [comment]
[](https://user-images.githubusercontent.com/82315023/161463804-8b484a81-c971-4141-84df-51d02f1d41be.png)

Overall the UI looks awesome.

Just a suggestion for the badge with icon component.

You can consider giving the icons a background color for better aesthetics.

**Review by athar05 — APPROVED** [review]
Overall the UI looks awesome.

Just a suggestion for the badge with icon component.

You can consider giving the icons a background color for better aesthetics.

image

File: styles/card.css [hunk 1]
```diff
@@ -62,12 +62,17 @@
...
+        /* bottom: 27.3rem;
+        left: 0.8rem; */
         position: relative;
-        bottom: 27.3rem;
-        left: 0.8rem;
-        background: transparent;
+        bottom: 36.5rem;
+        left: 1rem;
+        border: 1px solid var(--black);
+        /* background: transparent; */
```
Comment (rishichourasia) [inline styles/card.css:75]: You can remove unnecassary comment on final commit

**Comment by rishichourasia** [comment]
Seems good to go for me , small UI changes can be done like left aligning content on the navbar , text font seems off to me . overall theming seems dull . some colour theme changes can make huge difference .

**Pull request merged**